teos/src/main.rs

```diff
@@ -115,6 +115,20 @@ async fn main() {
         eprintln!("Cannot create network dir: {e:?}");
         std::process::exit(1);
     });
+
+    // Log default data dir
+    log::info!("Default data directory: {:?}", &path);
+
+    // Log config file path
+    log::info!("Config file: {:?}", &conf_file_path);
```

Comment:
This should log depending on whether the config file is found or not, so:

```
if is_default {
    log::info!("Config file: {:?} (not found, skipping)", &conf_file_path);
} else {
    log::info!("Config file: {:?}", &conf_file_path);
}
```

Reply:
You can also remove the previous log (L106-L110) where we were reporting whether we were loading data from the config file or not, that won't be necessary anymore.